modules/new_serverpod_auth/serverpod_auth_profile/serverpod_auth_profile_server/lib/src/models/user_profile.spy.yaml

```diff
@@ -0,0 +1,30 @@
+class: UserProfile
+table: serverpod_auth_profile
+fields:
+  id: UuidValue?, defaultPersist=random
+
+  ### The [AuthenticationUser]'s id.
+  userId: UuidValue, relation(parent=serverpod_auth_user)
+
+  ### The first name of the user or the user's nickname.
+  userName: String?
+
+  ### The full name of the user.
+  fullName: String?
+
+  ### The email of the user.
+  email: String?
```

Comment:
In `serverpod_auth` and current `serverpod_auth_email_account` we store the email as lowercase for later look up.

TIL that technically this is not correct, but in practice just fine: https://blog.teknkl.com/email-addresses-case-sensitive-but-anyway/

We should probably enforce the same rule here (on the model layer, not just convention), so that the look-up actually works case-insensitively.

Reply:
I checked back, and currently it's all lower case storage and lookup.

That probably makes the most sense going forward as well, as email addresses are practically case-insensitive.

With ILIKE we have to be careful with the wildcards, of which "_" is one, which is also part of email addresses. Will have to check how they behaves through our query layer.

Ideally I would create an index on the lower case variant of the email field, training the original, but I don't think that this is an option right now... and using "LOWER(email) = $email" might be slow at runtime (full table scan).